Update developer guide

dylkaw opened the pull request:
- Closes #70 
- Add use case for viewing members of event
- Add use case for viewing events of member
- Update non-functional requirements

dylkaw committed:
Add use case for viewing members of event

dylkaw committed:
Add use case for viewing events of member

dylkaw committed:
Add non-functional requirements for event numbers

wr1159 commented on docs/DeveloperGuide.md:
```diff
@@ -350,13 +350,68 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+
+      Use case resumes at step 2.
+
+**Use case: UC04 - View events of member**
```
I believe this should be UC05?

```suggestion
**Use case: UC05 - View events of member**
```

wr1159 reviewed (changes requested):
Left a small comment regarding Usecase ID, should be okay after changing it.

dylkaw committed:
Update numbering for viewing events of member

There were two use cases labelled UC04.

Hence Viewing Event of Member was renumbered UC05.

dylkaw committed:
Remove empty list extensions

Behaviour of command when list is empty should not be an extension.

wr1159 commented:
LGTM
